packages/wonder-blocks-link/__snapshots__/custom-snapshot.test.js.snap

```diff
@@ -136,45 +99,10 @@ exports[`LinkCore kind:primary href:# light:false pressed 1`] = `
 </a>
 `;
 
-exports[`LinkCore kind:primary href:# light:true disabled 1`] = `
```

Comment:
There's no such thing as a disabled link.

Reply:
It's called a "dimmed" link in assistive tech worlds and is created by giving it a tabIndex="-1" and adding `aria-disabled` as `true`. You can see an example here in webapp (https://github.com/Khan/webapp/blob/ee1ad038b3467ea09822745ae43fce6c0cfee88b/javascript/test-prep-package/practice-plan/components/checkpoint-progress.jsx#L230-L242)